openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -195,7 +195,7 @@ def schedules_for_bin(self):
             except InvalidContextError:
                 continue
 
-            yield (user, first_schedule.enrollment.course.language, template_context)
+            yield (user, first_schedule.enrollment.course.closest_released_language, template_context)
```

Comment:
Sanity check: what happens if `closest_released_language` is `None`?

Reply:
Here's where this value eventually gets used: https://github.com/edx/edx-ace/blob/1040bdd630caf76270b113f6a4a62c7fb43044b4/edx_ace/presentation.py#L26

If it's `None`, we eventually use `django.utils.translation.get_language()` instead.